gulpfile.js

```diff
@@ -147,6 +148,11 @@ gulp.task('deploy', gulp.series('build', function (done) {
     sign.on('exit', function (code, signal) {
       log('Signing done')
 
+      if (!doPublish) {
+        done()
```

Comment:
Maybe also notify the user that process is finished and publishing is skipped:

```suggestion
        log('Skipping publishing')
        done()
```

Reply:
Looks good, thanks!